### PR TITLE
fix: update not supported plugins errors

### PR DESCRIPTION
**Pull request by estrada9166**
- Closes  [UNIFY-1524](https://cypress-io.atlassian.net/jira/software/projects/UNIFY/boards/20/backlog?assignee=5ea883deddd45f0b94f5d98d&selectedIssue=UNIFY-1524)

### User facing changelog

Some plugins are not supported in v-10, is for this reason that the error that is being thrown has been updated depending if we are able to identify that the reason is the invalid plugin

### Before
image

### After
image

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

**Review by lmiller1990 — APPROVED** [review]
Definitely a big improvement. We still show the original error "glob pattern not found" but I'm guessing this is somewhere deep in driver and not practical to change.

Did we see if any other (official) plugins are broken by 10.x?

File: packages/driver/src/cypress/runner.ts [hunk 1]
```diff
@@ -987,6 +987,8 @@ const _runnerListeners = (_runner, Cypress, _emissions, getTestById, getTest, se
       hookName = getHookName(runnable)
       const test = getTest() || getTestFromHookOrFindTest(runnable)
 
+      const unsupportedPlugin = runnable.invocationDetails && runnable.invocationDetails.originalFile && runnable.err && runnable.err.message ? $errUtils.getUnsupportedPlugin(runnable.invocationDetails.originalFile, runnable.err.message) : null
```
Comment (chrisbreiding) [inline packages/driver/src/cypress/runner.ts:990]: Maybe pass the `runnable` to `$errUtils.getUnsupportedPlugin` and let it take care of this logic and return null if these properties aren't available?

**Review by chrisbreiding — CHANGES_REQUESTED** [review]
I think it would be better if the error message were formatted like:

```
Cypress detected that the current version of @cypress/code-coverage is not supported. Update it to the latest version.

The following error was caught:

> glob pattern string required

Because this error occurred during a after all hook we are skipping all of the remaining tests.
```

**Comment by lmiller1990** [comment]
Nice @estrada9166, for completion sake (and it it's not too difficult) can you add a screenshot of how the error looks? I think it'll be useful for historical context, if we ever look back, as well as for the final review and :heavy_check_mark:  :pray:

**Review by chrisbreiding — APPROVED** [review]

**Comment by lmiller1990** [comment]
Alright got the double ✅ , let's ship it.

**Pull request merged**